EssentialApp/EssentialAppTests/FeedImageCommentsAcceptanceTests.swift

```diff
@@ -0,0 +1,107 @@
+//
+//  FeedImageCommentsAcceptanceTests.swift
+//  EssentialAppTests
+//
+//  Created by Ivan Ornes on 7/4/21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import XCTest
+import EssentialFeed
+import EssentialFeediOS
+@testable import EssentialApp
+
+class FeedImageCommentsAcceptanceTests: XCTestCase {
+	
+	func test_onLaunch_displaysFirstFeedImageCommets() {
```

Comment:
It shouldn't show comments on launch. But on selecting an image.

Rename the test:

```suggestion
	func test_onImageSelection_displaysImageComments() {
```